lib/azure/core/http/http_request.rb

```diff
@@ -147,10 +140,7 @@ def body=(body)
         # @return [HttpResponse]
         def call
           conn = http_setup
-          res = conn.run_request(method.to_sym, uri, nil, nil) do |req|
-            req.body = body if body
-            req.headers = headers if headers
-          end
+          res = set_up_response(method.to_sym, uri, conn, headers ,body)
 
```

Comment:
In method set_up_response `req.options[:timeout] = keep_alive.split('=').last.to_i unless keep_alive.nil?`

Should timeout be read_timeout? I am curious as i am not seeing timeout in the [http://ruby-doc.org/stdlib-2.3.1/libdoc/net/http/rdoc/Net/HTTP.html#method-i-read_timeout-3D](http://ruby-doc.org/stdlib-2.3.1/libdoc/net/http/rdoc/Net/HTTP.html#method-i-read_timeout-3D) or it is it the same?